### DISCUSSION TITLE
How to set widget before shortcut icons?

### DISCUSSION THREAD

**Discussion by rankokrneta** [Lissy93/dashy#1204] [Help and Support] [answered]
Hi,

Does anyone can help me, how to show widget before shortut icons on dashboard?
![image](https://github.com/Lissy93/dashy/assets/77792967/f8abb680-f008-45eb-ba48-3f6b05280735)

@Lissy93, this is amazing work! Thank you!

**Comment by lissy93** [comment, marked as answer]
Good question, unfortunatey this isn't yet supported :(

But there's a couple options:
1. Add widgets to their own section
2. Reverse the order of a section, with CSS
   - `#section-outer-home-section-0 .content-inner { display: flex; flex-direction: column-reverse; }`
   - ^^ Replace `#section-outer-home-section-0` with the ID of your section

**Reply by rankokrneta** [reply]
_"#section-outer-home-section-0 .content-inner { display: flex; flex-direction: column-reverse; }"_
So smart! :D Bravo! 👍 
![image](https://github.com/Lissy93/dashy/assets/77792967/45b83d67-1c85-45ac-9ccd-d443e1e69bdf)
Thank you for fast response! 


p.s. I'm still just noob, just only one week into learn all great stuff in Dashy :)
